libvuln/migrations/migration1.go

```diff
@@ -54,6 +54,7 @@ const (
 		normalized_severity    TEXT,
 		package_name           TEXT,
 		package_version        TEXT,
+		package_module         TEXT,
```

Comment:
think this needs to be added to the index as it will be queried.

Reply:
Fixed